packages/design-system/src/__generated__/figma-design-tokens.ts

```diff
@@ -493,28 +535,31 @@ export const color = {
   foregroundOverwrittenFlexUi: "#dc3d43",
   foregroundInversePrimary: "#ff22ae",
   foregroundLinkNotVisited: "#006adc",
+  foregroundLinkHoverMain: "#656869",
```

Comment:
why is this called foregroundLinkHoverMain and not just foregroundLinkHover?

Reply:
When text color is foreground.main, text links should be foreground.link-hover.main

This is the pattern for all of these colors

Reply:
is main the default? or when is it usable

Reply:
Main is the default foreground color against light backgrounds, yes. It's the near-black color.

Reply:
if main is the default color, then there mist be also " foregroundLinkMain"

Reply:
honestly I don't understand the entire system here

Reply:
We need a color change when a text link is hovered. A text link is the same color as the rest of the text, so links do not need a different default color, only a different hover color.

So I identified the colors that are used for text in our UI, and made a hover color for each of them.

When text is foreground.main, links will also be foreground.main and links in the hover state will be foreground.link-hover.main 